app/decorators/reviewing_asset.rb

```diff
@@ -5,6 +5,11 @@ def review!
     save
   end
 
+  def unreview!
+    self.reviewed = false
+    save
```

Comment:
Should this be `save!`?  Or maybe the method should be `unreview` without the bang...?